api/plan.go

```diff
@@ -61,3 +61,12 @@ func removePlan(w http.ResponseWriter, r *http.Request, t auth.Token) error {
 	}
 	return err
 }
+
+func listRouters(w http.ResponseWriter, r *http.Request, t auth.Token) error {
```

Comment:
thanks @andrewsmedina 
there is already a router.go file in the api package but it's a file used for registering the handlers with Gorilla Mux router (or some stuff like that ..). 

https://github.com/tsuru/tsuru/blob/master/api/router.go

If I put there it will be strange, no? Is it ok if I rename this existent router.go(with delayedRouter stuff) file to delayed_router.go and just use the router.go file for the listRouters handler?